tests/integ/test_tfs.py

```diff
@@ -32,16 +33,19 @@ def tfs_predictor(instance_type, sagemaker_session, tf_full_version):
     model_data = sagemaker_session.upload_data(
         path='tests/data/tensorflow-serving-test-model.tar.gz',
         key_prefix='tensorflow-serving/models')
-    with timeout_and_delete_endpoint_by_name(endpoint_name, sagemaker_session):
+    with tests.integ.timeout.timeout_and_delete_endpoint_by_name(endpoint_name, sagemaker_session):
         model = Model(model_data=model_data, role='SageMakerRole',
                       framework_version=tf_full_version,
                       sagemaker_session=sagemaker_session)
         predictor = model.deploy(1, instance_type, endpoint_name=endpoint_name)
         yield predictor
 
 
-# @pytest.mark.continuous_testing
-# @pytest.mark.regional_testing
+@pytest.mark.continuous_testing
+@pytest.mark.regional_testing
+@pytest.mark.skipif(
+    tests.integ.REGION in tests.integ.HOSTING_P3_UNAVAILABLE_REGIONS,
+    reason='no ml.p3 instances in this region')
```

Comment:
I am not sure on this part. Will it skip c5 as well?

Reply:
oops yes. will fix